bindings/python/convert.py

```diff
@@ -95,18 +97,20 @@ def convert_multi(model_id: str, folder: str) -> List["CommitOperationAdd"]:
     operations = [
         CommitOperationAdd(path_in_repo=local.split("/")[-1], path_or_fileobj=local) for local in local_filenames
     ]
+    errors: List[Tuple[str, "Exception"]] = []
```

Comment:
It's Python, no need for the type hint here, especially since the name is descriptive enough :-)

Reply:
Ahah, I have mypy setup here, and it's not smart enough to figure it out on its own (and does yell if type is not present).